generator/internal/scaffold_generator.cc

```diff
@@ -261,10 +261,64 @@ void GenerateReadme(std::ostream& os,
 :construction:
 
 This directory contains an idiomatic C++ client library for
-[$title$](https://cloud.google.com/$site_root$/), a service that
-$description$
+[$title$][cloud-service-docs], a service that $description$
 
 This library is **experimental**. Its APIS are subject to change without notice.
+
+Please note that the Google Cloud C++ client libraries do **not** follow
+[Semantic Versioning](https://semver.org/).
+
+## Supported Platforms
+
+* Windows, macOS, Linux
+* C++11 (and higher) compilers (we test with GCC >= 5.4, Clang >= 6.0, and
+  MSVC >= 2017)
+* Environments with or without exceptions
+* Bazel and CMake builds
+
+## Documentation
+
+* Official documentation about the [$title$][cloud-service-docs] service
+* [Reference doxygen documentation][doxygen-link] for each release of this
+  client library
+* Detailed header comments in our [public `.h`][source-link] files
+
+[cloud-service-docs]: https://cloud.google.com/$site_root$
+[doxygen-link]: https://googleapis.dev/cpp/google-cloud-$library$/latest/
+[source-link]: https://github.com/googleapis/google-cloud-cpp/tree/main/google/cloud/$library$
+
+## Quickstart
+
+The [quickstart/](quickstart/README.md) directory contains a minimal environment
+to get started using this client library in a larger project. The following
+"Hello World" program is used in this quickstart, and should give you a taste of
+this library.
+
+<!-- inject-quickstart-start -->
+<!-- inject-quickstart-end -->
+
+* Packaging maintainers or developers that prefer to install the library in a
```

Comment:
s/that/who/ perhaps?

Reply:
Done here and everywhere else (maybe I went too far, let me know).